teerex/src/lib.rs

```diff
@@ -551,8 +546,8 @@ impl<T: Config> Pallet<T> {
 	fn verify_report(
 		sender: &T::AccountId,
 		ra_report: Vec<u8>,
-	) -> Result<SgxReport, DispatchErrorWithPostInfo> {
-		let report = verify_ias_report(&ra_report)
+	) -> Result<sgx_verify::SgxReport, DispatchErrorWithPostInfo> {
+		let report = sgx_verify::verify_ias_report(&ra_report)
 			.map_err(|_| <Error<T>>::RemoteAttestationVerificationFailed)?;
```

Comment:
There were unused imports with the `skip-ias-check` feature flag. Instead of adding a `#[cfg(feature ..)]` I inlined the imports as I found it more handy.